### PR TITLE
update README to use 0.5.0

### PR DESCRIPTION
**Pull request by Lendemor**
Update README.md

### CODE REVIEW

**Review by masenf — DISMISSED** [review]
i'm not 100% sure we want to show the experimental features right on the readme.

@picklelo what do you think?

**Comment by picklelo** [comment]
Agreed - discussed this with Thomas in our meeting today, let's remove the toast and keep the window alert for now

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ class State(rx.State):
     def get_image(self):
         """Get the image from the prompt."""
         if self.prompt == "":
-            return rx.window_alert("Prompt Empty")
+            return rx._x.toast.error("Prompt Empty")
```
Comment (picklelo) [inline README.md:103]: let's revert this

File: README.md [hunk 2]
```diff
@@ -185,15 +186,15 @@ class State(rx.State):
 
 The state defines all the variables (called vars) in an app that can change and the functions that change them.
 
-Here the state is comprised of a `prompt` and `image_url`. There are also the booleans `processing` and `complete` to indicate when to show the circular progress and image.
+Here the state is comprised of a `prompt` and `image_url`. There are also the booleans `processing` and `complete` to indicate when to block the button and when to show the image.
```
Comment (picklelo) [inline README.md:189]: Maybe "disable the button" rather than block

**Review by picklelo — CHANGES_REQUESTED** [review]

**Comment by tgberkeley** [comment]
When this PR is merged in I can do a follow up PR to update the image so that it exactly matches the code

File: README.md [hunk 3]
```diff
@@ -185,15 +186,15 @@ class State(rx.State):
...
+Here the state is comprised of a `prompt` and `image_url`. There are also the booleans `processing` and `complete` to indicate when to disable the button (during image generation) and when to show the resulting image.
 
 ### **Event Handlers**
 
 ```python
 def get_image(self):
     """Get the image from the prompt."""
     if self.prompt == "":
-        return rx.window_alert("Prompt Empty")
+        return rx._x.toast.error("Prompt Empty")
```
Comment (tgberkeley) [inline README.md:197]: You might want to change this back as well so that it uses the window alert instead of the experimental toast

Comment (Lendemor) [reply README.md]: done

**Review by picklelo — APPROVED** [review]

**Pull request merged**